### ISSUE TITLE
Handle fragments that are defined separately from use

### ISSUE THREAD

**Issue by lorensr** [apollographql/eslint-plugin-graphql#215]
Linting the below file results in errors:

```
   4:3   error  Fragment "ReviewEntry" is never used  graphql/template-strings
  22:10  error  Unknown fragment "ReviewEntry"        graphql/template-strings
```

Is there a way to either:

A. support this format of using fragments, or 
B. turn off just this rule?

```js
import gql from 'graphql-tag'

export const REVIEW_ENTRY = gql`
  fragment ReviewEntry on Review {
    id
    text
  }
`

export const ON_REVIEW_CREATED_SUBSCRIPTION = gql`
  subscription onReviewCreated {
    reviewCreated {
      ...ReviewEntry
    }
  }
  ${REVIEW_ENTRY}
`
```

**Comment by jnwng** [comment]
i go into more detail [here](https://github.com/apollographql/eslint-plugin-graphql/issues/226#issuecomment-493706108) but has to due with not being able to statically analyze the interpolated fragment. 

 the resolution here may be to ignore the `NoUsedFragment` rule, but the only way to configure this right now is to [safelist the list of validators you want to use](https://github.com/apollographql/eslint-plugin-graphql#selecting-validation-rules). 

addressing #98 should make this more straightforward.
